hack/make-rules/Makefile.manifest

```diff
@@ -1,30 +0,0 @@
-# Copyright 2018 The Kubernetes Authors.
```

Comment:
The pause image was the last actual consumer of this `Makefile.manifest` no longer has an results in this repo.